### DISCUSSION TITLE
kangaroo/faq

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2102] [Announcements]
# kangaroo/faq

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kangaroo/faq.html

**Comment by liyuerich** [comment]
scanner [ / ]$ trivy -h | grep 'TRIVY_CACHE_DIR' --cache-dir value  cache directory (default: "/root/.cache/trivy") [$TRIVY_CACHE_DIR]
bash: syntax error near unexpected token `('

**Reply by liyuerich** [reply]
复制 后直接运行，就是这个结果。 应该 只复制  trivy -h | grep 'TRIVY_CACHE_DIR'

**Comment by liyuerich** [comment]
trivy 离线数据库下载后，怎么导入 harbor呢？ 能不能给个说明？

**Comment by windsonsea** [comment]
@lengrongfu @lemontrees123
